### PR TITLE
Sei devnet 3 genesis

### PR DESCRIPTION
**Pull request by BrandonWeng**
Genesis file for sei-devnet-3 

* test accounts for loadtest client
* mint schedule
* vesting account
* faucet account
* see params https://github.com/sei-protocol/testnet/commit/f44c11e826a95be1862356528ad390590d2eabd9

### CODE REVIEW

**Review by yzang2019 — APPROVED** [review]

**Review by codchen — APPROVED** [review]

File: sei-devnet-3/genesis.json [hunk 1]
```diff
@@ -0,0 +1,16831 @@
...
+            "message_delay": "12000000000"
+        },
+        "timeout": {
+            "propose": "1000000000",
+            "propose_delta": "500000000",
+            "vote": "50000000",
+            "vote_delta": "500000000",
+            "commit": "50000000",
+            "bypass_commit_timeout": false
+        },
```
Comment (philipsu522) [inline sei-devnet-3/genesis.json:34]: this is ignored right?

File: sei-devnet-3/genesis.json [hunk 2]
```diff
@@ -0,0 +1,16831 @@
...
+                    "address": "sei1qv0ghcq442tvtxnztfktpvrccxwfwd3x78qq9g",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1atxjf0tsp8uysf2e3vzptu7qmsqv324flx0zxq",
+                    "pub_key": null,
+
```
Comment (philipsu522) [inline sei-devnet-3/genesis.json:7253]: there's 1k+ accounts - is that intentional?

File: sei-devnet-3/genesis.json (hunk 1 above)
Comment (BrandonWeng) [reply sei-devnet-3/genesis.json:34]: yep!

File: sei-devnet-3/genesis.json (hunk 2 above)
Comment (BrandonWeng) [reply sei-devnet-3/genesis.json:7253]: yeah, for loadtest clients

**Review by philipsu522 — APPROVED** [review]

**Pull request merged**